torch/csrc/autograd/custom_function.h

```diff
@@ -56,10 +56,13 @@ TORCH_API variable_list _wrap_outputs(
 // auto y = MyFunction::apply(6, x);
 // Example backward call:
 // y[0].sum().backward();
+template<typename X, typename... Args>
+using forward_t = decltype(X::forward(nullptr, std::declval<Args>()...));
```

Comment:
You probably didn't mean to attach the docblock above to `forward_t`. (Speaking of which, would be nice to have a doc for `forward_t` too!)